mpi-proxy-split/record-replay.h

```diff
@@ -453,9 +470,11 @@ namespace dmtcp_mpi
 	    {
               MPI_Datatype type = rec->args(0);
 	      datatype_free(type);
-	      break;
+	      delete rec;
+	      return NULL;
             }
-	    default: break;
+	    default:
```

Comment:
Please add the following comment for this `default:` label:
```
	      // The 'default' cases include record types like
              //     comm_create, comm_group, group_incl, etc. 
	      // Those known types only need to be recorded.  So they don't
              //     have any case label to pre-process their record info
              //     before they are recorded.
```

Reply:
I have added the above comments for the `default` label. Thanks Gene!